frameworks/PHP/hamlet/benchmark_config.json

```diff
@@ -47,26 +46,22 @@
       "notes": "",
       "versus": "php"
```

Comment:
Please change to:
`"versus": "swoole"`

So we will see the framework overhead correctly.
Check:
https://www.techempower.com/benchmarks/#section=test&runid=9f738614-61d5-4c04-a1a5-3778b2ac3003&hw=ph&test=fortune&l=zik073-7
and click the tab `Framework overhead`